### ISSUE TITLE
Error in install google-chrome-dev

### ISSUE THREAD

**Issue by specter119** [caskroom/homebrew-cask#21838]
## Installation
I typed `brew cask install google-chrome-dev`
and got error message like:
```
=> Caveats
The Mac App Store version of 1Password won't work with a Homebrew-Cask-linked Google Chrome. To bypass this limitation, you need to either:

  + Move Google Chrome to your /Applications directory (the app itself, not a symlink).
  + Install 1Password from outside the Mac App Store (licenses should transfer automatically, but you should contact AgileBits about it).

==> Downloading https://dl.google.com/chrome/mac/dev/googlechrome.dmg
######################################################################## 100.0%
==> No checksum defined for Cask google-chrome-dev, skipping verification
Error: It seems the App source is not there: '/opt/homebrew-cask/Caskroom/google-chrome-dev/latest/Google Chrome.app'
```
when i open the tmp dmg, only a README.txt in it, which shows:
```
This disk image contains a differential updater that can update
Google Chrome from version 53.0.2756.0 to 53.0.2763.0 Dev.

This image is part of the auto-update system and is not independently
useful.

To install Google Chrome, please visit
<http://www.google.com/chrome/>.
```

## Update:
Later downloaded it again from the offical site, and got the same dmg, it seems that there isn't a full installation dmg for chrome-dev.

**Comment by vitorgalvao** [comment]
Thank you for the report. Not much we can do here but remove the cask. Will do so shortly.

**Comment by maxnordlund** [comment]
Strange, when I moved my caskroom today as prompted by #21913 I noticed that this cask got removed. But when I tried to download the cask manually using both the official website and wget, it got me the full version. So my guess is that it was a hiccup on Googles side. The only difference seems to be casing but wget could download the lower case one just fine.

https://dl.google.com/chrome/mac/dev/GoogleChrome.dmg
https://dl.google.com/chrome/mac/dev/googlechrome.dmg
